### ISSUE TITLE
Seperate flowkit into its own repo

### ISSUE THREAD

**Issue by bjartek** [onflow/flow-cli#655]
Right now flowkit is not represented as a tag but just a hash. Since flowkit is a core component in the way other go projects integrate with flow it should be released undrer semantic versioning and be in its own repository.

**Comment by sideninja** [comment]
I agree. Currently, the flowkit is just replaced in the go.mod for the CLI. Our plan is to extract flowkit like you mentioned, but it will take a bit of a time to do that work, as we are now working on something more critical. During that process I want to also take in all the feedback of making the flowkit work like a pure library as we discussed and do all the work dedicated to flowkit.

**Comment by bjartek** [comment]
Just a little heads up on this issue, the recent  0.45.x branch of flow-cli contains breaking changes for flowkit. If others use it and faced the same problems i did.

**Comment by sideninja** [comment]
Closing this due to https://github.com/onflow/flow-cli/issues/856 
I think extracting to a separate repo is not really needed, but we should start doing proper tagging, which would solve issues described.

**Comment by bjartek** [comment]
Yeah the main issue is really tagging. not seperate repo.
